actors/market/src/lib.rs

```diff
@@ -751,15 +748,15 @@ impl Actor {
                         },
                     ));
                     sector_deal_ids.push(deal_id);
-                    pieces_ret.push(ext::miner::PieceReturn { accepted: true });
+                    ret.accepted = true;
```

Comment:
see https://github.com/filecoin-project/builtin-actors/pull/1500 for a couple of lines that were needed to make this work but were left out somehow (it looks like you had it in there but they fell out somehow before commit?)